baricadr/model/backends.py

```diff
@@ -82,12 +82,12 @@ def remote_is_single(self, repo, path):
         Check if distant path is a single file or not
         """
 
-        remote_list = self.remote_list(repo, path)
+        remote_list = self.remote_list(repo, path, max_depth=0)
         return len(remote_list) == 1
 
     # TODO expose remote_list in api ?
     # TODO we could use the --hash option of lsjson (may be slow, but may be useful)
-    def remote_list(self, repo, path, full=False):
+    def remote_list(self, repo, path, full=False, compare=False, max_depth=1):
```

Comment:
rename `compare` to `remote_only` or something clearer?

Reply:
How about `missing` ? Not sure remote_only is clearer

Reply:
:+1: